raiden-ts/tests/unit/epics/send.spec.ts

```diff
@@ -87,7 +88,9 @@ describe('send transfer', () => {
       lock: {
         secrethash,
         amount: value.add(fee),
-        expiration: expect.any(BigNumber),
+        expiration: BigNumber.from(Math.ceil(1.1 * raiden.config.revealTimeout))
+          .add(requestBlock)
+          .sub(1),
```

Comment:
I don't really get why I need to subtract one here.

Reply:
Hmm, me neither.. can you debug `requestBlock` against the current `state.blockNumber`?